src/Directory.Build.props

```
@@ -2,7 +2,7 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;net6.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>net6.0;net462</TargetFrameworks>
```

for consistency when using VSCode

---

https://github.com/signalfx/splunk-otel-dotnet/pull/94/commits/f95edd8007d5284fcb0a6ebfece3fc222f19046c